packages/dd-trace/test/exporters/agent/exporter.spec.js

```diff
@@ -72,4 +74,16 @@ describe('Exporter', () => {
       expect(writer.flush).to.have.been.called
     })
   })
+
+  describe('setUrl', () => {
+    beforeEach(() => {
+      exporter = new Exporter({ url })
+    })
+    it('should set the URL on self and writer', () => {
```

Comment:
This tests the code and not the behaviour. It should test that the URL is actually used to make the request.

Reply:
There's now a setUrl method on writer, which this calls. That method is tested to impact the flush URL.